packages/api-catalog-service/service.ts

```diff
@@ -0,0 +1,88 @@
+import { ApolloServer } from 'apollo-server-express';
+import express from 'express';
+import { mergeSchemas } from 'graphql-tools';
+import http from 'http';
+import mongoose from 'mongoose';
+import { NamespaceResolver } from './src/namespace/resolver';
+import NamespaceSchema from './src/namespace/typedef.graphql';
+import { NSNotificationResolver } from './src/notifications/resolver';
+import NSNotificationSchema from './src/notifications/typedef.graphql';
+import SharedSchema from './src/shared/typedef.graphql';
+
+import cookieParser = require( 'cookie-parser' );
+
+import { scheduleJob } from 'node-schedule';
```

Comment:
Use `agenda` instead of node-schedule. node-schedule causes a lot of duplicate runs when the app has multiple replicas.

Reply:
updated